package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "leaflet.functionaltilelayer",
-  "version": "0.0.0",
+  "version": "1.0.0",
   "description": "Leaflet tile layer with functionally defined URL and support for promises.",
-  "keywords": ["leaflet"],
-  "main": "src/leaflet.functionaltilelayer.js",
+  "keywords": [
+    "leaflet"
+  ],
+  "main": "./src/leaflet.functionaltilelayer.js",
```

Comment:
The relative path that was there previously should have worked. Was it breaking for you in a particular case?